packages/react-core/src/components/Card/examples/Card.md

```diff
@@ -68,6 +68,10 @@ import pfLogoSmall from './pf-logo-small.svg';
 
 ### Selectable
 
+Note: The listbox role on the containing div of this examples is needed because the cards have the option role via their `isOption` prop, and options are required to be inside a listbox. See the [option role documentation](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/option_role) from MDN for more information.
+
+The cards need to be options in order to effectively communicate that they're selectable (and what their current selection state is) for those using screen readers.
+
```

Comment:
**Nit:** The final line could maybe be updated from "for those using screen readers" to "for those using assistive technologies (such as screen readers)." Train of thought is mainly inclusivity.

**General comment** that maybe @nicolethoen could weigh in on: Could it be worth noting that adding the `role="option"` to the add card button *isn't* best practice since it conveys that the card is selectable when it isn't?

Reply:
Good callout 🔥